files/src/utils/test-helpers/test-helper.ts

```diff
@@ -0,0 +1,7 @@
+import { setApp, start } from '@glimmer/test-helpers';
+import App from '../../main';
+
+QUnit.config.autostart = false;
```

Comment:
I'm pretty sure we can if we explicitly use that in the `ember-cli-build.js`, but can we make that happen automatically? I suppose we'd have to create our own npm package wrapping npmjs, which is a shame.